jumpscale/servers/threebot/threebot.py

```diff
@@ -268,11 +269,15 @@ def bottle_servers(self):
 
     @property
     def actors(self):
-        return self._load_files(self.actors_dir)
+        if self.actors_dir:
+            return self._load_files(self.actors_dir)
```

Comment:
I think the return value is not consistent here as `self._load_files` would return a generator here?

Also, IMO, we can do this change in `self._load_files` instead of repeating it in every method which calls `_load_files`.